pages/common/mosquitto_pub.md

```diff
@@ -0,0 +1,27 @@
+# mosquitto_pub
+
+> A simple MQTT version 3.1.1 client that will publish a single message on a topic and exit.
+
+- Publish a temperature value of 32 on the topic `sensors/temperature` to `localhost` [default] with Quality of Service(`QoS`) set to 1:
+
+`mosquitto_pub -t {{sensors/temperature}} -m {{32}} -q {{1}}`
+
+- Publish timestamp and temperature data to a remote host on a non-standard port and `QoS` 0:
+
+`mosquitto_pub -h {{192.168.1.1}} -p {{1885}} -t {{sensors/temperature}} -m {{"1266193804 32"}}`
+
+- Publish light switch status. Message is set to retained because there may be a long period of time between light switch events:
+
+`mosquitto_pub -r -h "iot.eclipse.org" -t {{switches/kitchen_lights/status}} -m {{"on"}}`
+
+- Send the contents of a file (`data.txt`) as a message to a defined topic:
+
+`mosquitto_pub -t {{my/topic}} -f {{data.txt}}`
+
+- Send the contents of a file (`data.txt`), by reading from `stdin` and send the entire input as a message to a defined topic:
+
+`mosquitto_pub -t {{my/topic}} -s < {{data.txt}}`
+
+- Read newline delimited data from `stdin` as a message and publish it to `sensors/cc128` topic:
+
+`read_cc128.pl | mosquitto_pub -t {{sensors/cc128}} -l`
```

Comment:
What is "read_cc128.pl" ? Can we just write "{{echo data.txt}}" to match with the previous example ?

Reply:
Missed that, thanks.